docs/dev/logger.md

```diff
@@ -0,0 +1,52 @@
+### Logging
+
+Logging utilizes the [`github.com/cihub/seelog`](https://github.com/cihub/seelog) package as its underlying framework. 
+It is accessible via `pkg/util/log` and the `comp/core/log` component wrapper.
+Using the component wrapper is recommended, as it adheres to [component best practices](https://datadoghq.dev/datadog-agent/components/overview/).
+
+#### Writing a good log message
+
+In general, there are both a few rules and a few suggestions to follow when it comes to writing a
+(good) log message:
+
+- Messages must be written in English. No preference on which specific English dialect is used e.g.
+  American English, British English, Canadian English, etc.
```

Comment:
```suggestion
- Messages must be written in English. No preference on which specific English dialect is used, for example,
  American English, British English, or Canadian English.
```

Reply:
I'd probably opt that we remove the reference to dialect. If we keep the dialect comment, we may want to use `American English` since it would be more consistent with spelling of some key terms in the codebase (e.g. `Initialize`  with ~600+ occurrences vs `Initialise` with ~20 occurences)